api/src/client/AwsMessagingServiceClient.ts

```diff
@@ -0,0 +1,111 @@
+import { InvokeCommand, LambdaClient } from "@aws-sdk/client-lambda";
+import { HealthCheckMetadata, MessagingServiceClient } from "@domain/types";
+import { LogWriterType } from "@libs/logWriter";
+import { ReasonPhrases } from "http-status-codes";
+
+export const AwsMessagingServiceClient = (config: {
+  functionName: string;
+  logWriter: LogWriterType;
+}): MessagingServiceClient => {
+  const logId = config.logWriter.GetId();
+  const lambdaClient = new LambdaClient({});
+
+  const sendEmail = async (
```

Comment:
We can make this function name a little more generic to something like `sendMessage`. The Messaging Service will determine the channel (email, text, etc).

```suggestion
  const sendMessage = async (
```